test/system/flows/edit_solution_test.rb

```diff
@@ -0,0 +1,58 @@
+require "application_system_test_case"
```

Comment:
The react component is in the right place now, and is hitting the correct endpoints.